smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -555,6 +555,10 @@ contract MixinKeys is MixinErrors, MixinLockCore {
     uint formerTimestamp = _keys[_tokenId].expirationTimestamp;
 
     if (_addTime) {
+      // cant add to a non-expiring key
+      if (formerTimestamp == type(uint).max) {
```

Comment:
I am a bit worried about this change. Does that break anything?

Reply:
not that I know of. It just prevents from adding to max uint time. We can also not add it and just let the tx revert.